nodemon.json

```diff
@@ -1,6 +1,14 @@
 {
-  "watch": ["api/", "api-lib/", "scripts/", "src/lib", "src/common-lib"],
+  "watch": [
+    "api/",
+    "api-lib/",
+    "scripts/",
+    "src/lib",
+    "src/common-lib",
+    "*.json"
+  ],
   "delay": 300,
+  "verbose": true,
```

Comment:
can we turn this off? it's had this effect:

![Screen Shot 2022-10-19 at 3 24 00 PM](https://user-images.githubusercontent.com/193667/196624426-805c7465-8ef4-4b93-a92f-7478c2866d70.png)